prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/HL7ACKUtils.kt

```diff
@@ -0,0 +1,42 @@
+package gov.cdc.prime.router.fhirengine.translation.hl7.utils
+
+import ca.uhn.hl7v2.model.Message
+import ca.uhn.hl7v2.model.v251.message.ACK
+import gov.cdc.prime.router.common.Environment
+import gov.cdc.prime.router.fhirengine.utils.HL7Reader
+import java.time.Clock
+import java.util.Date
+import java.util.UUID
+
+/**
+ * Helper class to generate HL7 ACK response
+ */
+class HL7ACKUtils(
+    private val clock: Clock = Clock.systemUTC(),
+) {
+
+    fun generateOutgoingACKMessage(incomingACKMessage: Message): String {
+        val outgoingAck = ACK()
+
+        val ackMsh = outgoingAck.msh
+        ackMsh.msh1_FieldSeparator.value = "|"
+        ackMsh.msh2_EncodingCharacters.value = "^~\\&"
+        ackMsh.msh3_SendingApplication.parse("ReportStream")
```

Comment:
I took these values directly from the Spec document so just double check that these are exactly what we want.